BREAKING.md

```diff
@@ -39,6 +39,7 @@ IFluidResolvedUrl is now deprecated, all usages should move to IResolvedUrl inst
 -   [PureDataObject.getFluidObjectFromDirectory removed](#PureDataObject.getFluidObjectFromDirectory-removed)
 -   [IProvideContainerRuntime and IContainerRuntime member removed](#IProvideContainerRuntime-and-IContainerRuntime-member-removed)
 -   [IntervalCollection removed](#IntervalCollection-removed)
+-   [FluidDataStoreRuntime.getChannel throws for channels that do not exist](#FluidDataStoreRuntime.getChannel-throws-for-channels-that-do-not-exist)
```

Comment:
I think I should be creating a new 2.0.0-internal.6.0.0 section instead of adding on to the 5.0.0 section here since 5.0.0 is released, but all the packages are still on 5.0.0 in the `next` branch.  Probably just weird timing - will update the BREAKING accordingly if needed.

Reply:
Alternately, I see @tylerbutler has tagged this as needing a changeset and I think we maybe started using these for the client packages in 5.0.0?  In that case, should I ONLY add a changeset and not mess with BREAKING.md anymore?